z2/src/chain/node.rs

```diff
@@ -516,6 +516,20 @@ impl ChainNode {
         let eth_chain_id = self.eth_chain_id.to_string();
         let bootstrap_public_ip = selected_bootstrap.machine.external_address;
 
+        let contracts = self.chain()?.get_whitelisted_evm_contracts();
+        let whitelisted_evm_contract_addresses = if contracts.is_empty() {
+            String::from("[ ]")
+        } else {
+            format!(
```

Comment:
This code would be a lot cleaner if we can just pass a `serde_json::Value` into the `Tera` `Context`.

I have made a similar change here, which might be worth moving into this PR if you agree?